src/main/java/me/greenleaf/Main.java

```diff
@@ -0,0 +1,12 @@
+package me.greenleaf;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Ask Legolas a question: ");
+        String question = scanner.next();
+        System.out.print("Legolas says: " + Legolas.getInstance().queryMind(question));
+    }
```

Comment:
One can no longer attempt to lower `Legolas`' `SASS_LEVEL`. The program should either use stdin to ask the desired operation or require an argument to identify the operation desired.